### PR TITLE
Added new faster advanced tools for R&D to print.

### PR DESCRIPTION
**Pull request by lordpidey**
:cl:
rscadd: Nanotransen would love to proudly announce that we have upgraded all tools on the station.  Unfortunately, during the celebratory party, Steve from I.T. accidentally deleted the designs.  We are confident that your R&D department can re-create them if necessary
/:cl:.

### CODE REVIEW

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -99,6 +107,17 @@
...
+	name = "Advanced screwdriver"
+	desc = "Produces high powered micro vibrations to loosen stuck and stripped screws."
+	icon_state = "screwdriver_adv"
+	item_state = "screwdriver_blue"
+	toolspeed = 1.5
+
+/obj/item/weapon/screwdriver/advanced/New(loc)
+	if (prob(75))
+		src.pixel_y = rand(0, 16)
+
```
Comment (KazeEspada) [inline code/game/objects/items/weapons/tools.dm]: Needs a parent call on new.

**Comment by KazeEspada** [comment]
:+1:  for advanced tool
:-1:  since RnD will horde them all.

**Comment by Fox-McCloud** [comment]
Should probably add toolspeed to some of the more advanced welding tools.

**Comment by Buggy123** [comment]
:+1:
The advanced welder being faster would be nice too. Although the plasma cutter is already faster for cutting through walls...

**Comment by Bawhoppen** [comment]
:-1:x:100: R&D does not need more shit

if you want to give them to science, make it so they can be created by the experimentor only

**Comment by Fox-McCloud** [comment]
@Buggy123 

Unlocking/locking the fire axe cabinet =p

**Comment by Buggy123** [comment]
I stand corrected. :+1: give science more awesome shit.

**Comment by WJohn** [comment]
1.5x isn't nearly enough for me to care about upgrading to those. Go 2.0 or go home. That, or give these tools extra functions (crowbaring floor tiles crowbars many, increased melee damage etc).

Also a green stripe on the tools doesn't look terrible advanced.

**Comment by kevinz000** [comment]
... as much as I like R&D, this is seriously looking like a gimmick. 50% more speed isn't that much. Make it faster but way more expensive . Like a self charging powered tool

File: code/game/objects/items/weapons/tools.dm (hunk 1 above)
Comment (lordpidey) [inline code/game/objects/items/weapons/tools.dm]: Parent call will mess up the icon though, since the parent randomizes the icon.

**Comment by TheWiznard** [comment]
@WJohn what if the crowbar could crowbar like 1x3 or more area

**Comment by Iamgoofball** [comment]
yeah just make it 2x speed

also yeah increase the effects that way too

**Comment by GunHog** [comment]
Keep the speed differences in whole numbers, please. Other than that, :+1:

File: code/game/objects/items/weapons/tools.dm (hunk 1 above)
Comment (Jalleo) [inline code/game/objects/items/weapons/tools.dm]: Then have it override that in new.

**Comment by Bawhoppen** [comment]
why not make a multitool that prevents shocking from doorhacking (could do the same with wirecutters too)

**Comment by Fox-McCloud** [comment]
>why not make a multitool that prevents shocking from doorhacking (could do the same with wirecutters too)

Does every member of science need insulated gloves?

**Comment by lordpidey** [comment]
From the comments, it's clear that speed alone will not do for these tools, and I don't feel like coding new stuff for them, all of my major code time is going to demon.

**Pull request closed without merging**